code/game/machinery/doors/airlock.dm

```diff
@@ -1063,34 +872,26 @@ About the new airlock wires panel:
 	if(!forced)
 		if( !arePowerSystemsOn() || isWireCut(AIRLOCK_WIRE_OPEN_DOOR) )
 			return 0
-	use_power(50)
-	if(forced)
-		playsound(src.loc, 'sound/machines/airlockforced.ogg', 30, 1)
-	else if(istype(src, /obj/machinery/door/airlock/glass))
+	use_power(360)	//360 W seems much more appropriate for an actuator moving an industrial door capable of crushing people
+	if(istype(src, /obj/machinery/door/airlock/glass))
 		playsound(src.loc, 'sound/machines/windowdoor.ogg', 100, 1)
 	else if(istype(src, /obj/machinery/door/airlock/clown))
 		playsound(src.loc, 'sound/items/bikehorn.ogg', 30, 1)
 	else if(istype(src, /obj/machinery/door/airlock/mime))
+		// Play nothing
 	else
 		playsound(src.loc, 'sound/machines/airlock.ogg', 30, 1)
 	if(src.closeOther != null && istype(src.closeOther, /obj/machinery/door/airlock/) && !src.closeOther.density)
 		src.closeOther.close()
-	// This worries me - N3X
-	if(autoclose  && normalspeed)

```

Comment:
Doesn't this bit of code (from 1078 to 1084) still need to be in somewhere to retain the behavior of airlocks closing very quickly, after opening (overridden safeties)?